package.json

```diff
@@ -78,6 +78,7 @@
     "react": "16.13.x"
   },
   "dependencies": {
+    "@emotion/react": "^11.1.5",
```

Comment:
Oh, I'm interested in why this got added - is it needed? I don't fully know the context, but I believe there's a conversation going on with the dotcom team about maybe moving to `@emotion/react` but I don't think that's happened yet. On DCR currently the `emotion` dependency which is shared with `@guardian/braze-components` comes from `import * as emotion from 'emotion'`. Maybe we should be using that?